Cargo.toml

```diff
@@ -38,3 +38,9 @@ rust-llvm = [
     "llvm-sys/disable-alltargets-init"
 ]
 default = ["rust-llvm"]
+
+[profile.release]
+debug = true
+
+[patch.crates-io]
+aya-rustc-llvm-proxy = { git = "https://github.com/aya-rs/rustc-llvm-proxy.git" }
```

Comment:
This needs to be removed before merging yeah?

Reply:
You tell me - would you like to cut a release, or should we just take a git dep on the thing?

Reply:
Pulled this out of this PR.